pkg/util/logging/log_counter_hook.go

```diff
@@ -17,22 +17,28 @@ limitations under the License.
 package logging
 
 import (
+	"errors"
+	"fmt"
 	"sync"
 
 	"github.com/sirupsen/logrus"
+
+	"github.com/vmware-tanzu/velero/pkg/util/results"
 )
 
 // LogCounterHook is a logrus hook that counts the number of log
 // statements that have been written at each logrus level.
 type LogCounterHook struct {
-	mu     sync.RWMutex
-	counts map[logrus.Level]int
+	mu      sync.RWMutex
```

Comment:
comment should be updated b/c with this change this is no longer only for counting the number of log messages.